editor.planx.uk/src/ui/shared/ErrorSummary.tsx

```diff
@@ -0,0 +1,36 @@
+import Box from "@mui/material/Box";
+import { styled } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import React from "react";
+
+interface Props {
+  heading?: string;
+  message?: string;
+  format?: "error" | "warning" | "info";
+}
+
+const Root = styled(Box, {
+  shouldForwardProp: (prop) => !["format"].includes(prop.toString()),
```

Comment:
the syntax of `shouldForwardProp` is still never quite intuitive to me, but I think this can simply be ??
```
  shouldForwardProp: (prop) => prop !== "format",
```

and then on L14 just below, might need to adjust to `<Props["format"]>` rather than current `<Props>`, as we're just forwarding along the single property?

Reply:
Thanks, this works without adjusting line 14 👍